src/components/pages/BeregnPage.tsx

```diff
@@ -8,7 +8,7 @@ import { FormContext } from '@/contexts/context'
 import LoadingComponent from '../LoadingComponent'
 import { produce } from 'immer'
 
-function fetchBeregnData(formState: State) {
+export function fetchBeregnData(formState: State) {
```

Comment:
Denne util functionen:
- bør ikke leve her på toppen av komponenten. Kunne den den flyttes under `/functions`?
- den gjør ikke så mye annet enn å kalle `submitForm` og jeg ser at hverken denne eller `submitForm` brukes mer enn én gang. Kunne den merges med `submitForm` i én og samme function istedet?
- den ser ikke ut til å ha med fetching å gjøre. Bør den få et bedre navn. Hvis den merges med `submitForm` kan den vel bare hete `submitForm` så slipper du problemstillingen med navngivning.

Reply:
Tror det kan være greit å flytte den til /functions. Litt usikker på hvorfor du mener at fetchBeregnData er et dårlig navn. For den rendrer wrapPromise med daten vi mottar fra submitForm. I wrapPromise så thrower vi en suspender når daten ikke er mottat enda, errors vil bli catchet av error boundary dersom det skulle forekomme, og til slutt dersom daten er mottat vil den bli returnert. Med andre ord så fetcher den j daten, vil du ikke si det?

Reply:
I dag står det slik: `fetchBeregnData` -> `wrapPromise` -> `submitForm`. Og det er her i sudbmit at selve fetching skjer. Det er i den forstand at navngivningen synes jeg ikke var optimal.
Fordi det er egentlig første ledd som er tett på grensesnittet som har med submit å gjøre. Så dypere i lagene skjer det fetching. (ikke omvendt som dagens navngivning tilsier) Jeg ville derfor heller bruke navn i motsatt rekkefølge: validateFields, handleSubmit/onSubmit, og til slutt fetchBeregnData/postData

I tillegg, i og med at `submitForm` kun ser ut til å brukes én gang. Kunne disse 3 funksjonene bli til én, og f.eks da hete `handleSubmit/onSubmit` og bo under /functions